lang/vi/pagination.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/*
+|--------------------------------------------------------------------------
+| Pagination Language Lines
+|--------------------------------------------------------------------------
+|
+| The following language lines are used by the paginator library to build
+| the simple pagination links. You are free to change them to anything
+| you want to customize your views to better match your application.
+|
+*/
+
+return [
+    'next'     => 'Trang trước &raquo;',
```

Comment:
```suggestion
    'next' => 'Trang trước &raquo;',
```
fix tương tự các chỗ khác nha